docs/user/filter-and-process/README.md

```diff
@@ -1,16 +1,31 @@
-- Configuring the data source (Istio `Telemetry` CRD): For Istio-generated data (access logs and traces), you configure the Istio `Telemetry` resource itself. This controls which workloads generate data and at what volume (sampling rate), before it is even sent to a pipeline.
+You can select or reject data at the source, before it enters a pipeline. This is useful for controlling data volume from specific namespaces, applications, or Kubernetes resources. This method is efficient as it prevents unwanted data from being processed at all.
 
-## Processing Data
+The Telemetry module supports the following input-level filtering mechanisms:
 
-Application logs from containers are automatically transformed into structured OpenTelemetry (OTLP) log records.
+- **Pipeline input filtering** (LogPipeline, MetricPipeline): Configure the `input` section of a pipeline to select or reject data before it is processed by the agent. This is the most common method for filtering application logs, runtime metrics, and Prometheus metrics.
+- **Source-level filtering** (Istio `Telemetry` CRD): For Istio-generated data (access logs and traces), configure the Istio `Telemetry` resource itself to control which workloads generate data and at what volume (sampling rate).
 
-All pipelines automatically enrich telemetry data with Kubernetes resource attributes, such as Pod name, namespace, and labels. With this context information, you can easily identify the source of telemetry data in your backend.
+For details, see [Filter Logs](filter-logs.md), [Filter Metrics](filter-metrics.md), and [Filter Traces](filter-traces.md).
+
+## Pipeline Processing with OTTL
+
+After data is collected into a pipeline, you can perform custom transformations and filtering using the OpenTelemetry Transformation Language (OTTL). This gives you fine-grained control to modify attributes, redact sensitive information, or drop data based on complex conditions.
+
+For details, see [Custom Pipeline Processing with OTTL](custom-pipeline-processing-ottl.md).
+
+## Automatic Processing
+
+In addition to these user-configurable methods, the Telemetry module also performs several automatic processing steps to ensure your data is structured and useful:
+
+- **Data transformation**: Application logs from containers are automatically transformed into structured OpenTelemetry (OTLP) log records.
+- **Data enrichment**: All pipelines automatically enrich telemetry data with Kubernetes resource attributes, such as Pod name, namespace, and labels, so you can easily identify the source of telemetry data in your backend.
+
+For details, see [Transformation to OTLP Logs](transformation-to-otlp-logs.md) and [Automatic Data Enrichment](automatic-data-enrichment.md).
```

Comment:
Also move the details into the listing, otherwise you need to think about which link is for which item in the listing.